src/main/scala/is/hail/variant/GenomeReference.scala

```diff
@@ -200,8 +203,9 @@ object GenomeReference {
   var defaultReference = GRCh37
 
   def addReference(gr: GenomeReference) {
-    if (references.contains(gr.name))
-      fatal(s"Cannot add reference genome. Reference genome `${ gr.name }' already exists.")
+    if (hasReference(gr.name))
+      fatal(s"Reference genome `${ gr.name }' already exists.")
```

Comment:
You need to add 'Cannot add reference genome.' here or remove from test to fix the test failure.